app.riftshare.RiftShare.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: app.riftshare.RiftShare
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
```

Comment:
does it really need the GNOME platform?

Reply:
Yeah unfortunately there is a dependency on libgtk3 and libwebkit for the frontend as required by the framework I'm using.

Reply:
gtk3 is in the Freedesktop-SDK. 

But webkit... indeed. that's probably needed.